3.2/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -259,3 +259,59 @@ function setup_keyfile() {
 function info() {
   printf "=> [%s] %s\n" "$(date +'%a %b %d %T')" "$*"
 }
+
+# insert_and_wait_for_replication insert data in host and wait all replset members
```

Comment:
The reason why I moved them into the image it, that running tests which access replicaset requires running it inside of kubernetes. So it have to be run in container (discussed above - https://github.com/sclorg/mongodb-container/pull/222#issuecomment-275657513), which also contains `mongo` client. And tested image itself seems to me to be the best candidate to run tests in.

Therefore I moved them to image itself, to avoid declaring them through `kubectl run`. And also to be easily re-run tests by somewhere else.

But if it seems confusing I can move them somewhere...

 >  can you move them to a common test util script instead

Do you this "common test util script" in image or only separated in own file?